subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsPlugin.kt

```diff
@@ -284,7 +284,8 @@ public class InstrumentationTestsPlugin : Plugin<Project> {
             kubernetesNamespace = configuration.kubernetesNamespace,
             targets = getTargets(configuration, mergedInstrumentationParameters),
             enableDeviceDebug = configuration.enableDeviceDebug,
-            timeoutInSeconds = configuration.timeoutInSeconds,
+            testRunnerExecutionTimeout = configuration.testRunnerExecutionTimeout,
+            gradleTaskTimeout = configuration.gradleTaskTimeout,
```

Comment:
`instrumentationTaskTimeout`
gradleTask is too general description

Reply:
✅